File: 2006Scape Server/src/main/java/com/rs2/integrations/discord/commands/Online.java
package com.rs2.integrations.discord.commands;

import com.rs2.Constants;
import org.javacord.api.entity.message.Message;
import org.javacord.api.event.message.MessageCreateEvent;
import org.javacord.api.listener.message.MessageCreateListener;

import com.rs2.integrations.discord.JavaCord;

public class Online implements MessageCreateListener {
    @Override
    public void onMessageCreate(MessageCreateEvent event) {
        Message message = event.getMessage();
        if (message.getContent().equalsIgnoreCase(JavaCord.commandPrefix + " online")) {
            event.getChannel().sendMessage(":tada: " + JavaCord.serverName + " World:" + Constants.WORLD + " is Online! :tada:");
        }
    }
}

File: 2006Scape Server/src/main/kotlin/com/rs2/integrations/discord/commands/Online.kt
package com.rs2.integrations.discord.commands

import com.rs2.GameConstants
import com.rs2.integrations.discord.JavaCord
import org.javacord.api.event.message.MessageCreateEvent
import org.javacord.api.listener.message.MessageCreateListener

class Online : MessageCreateListener {
    override fun onMessageCreate(event: MessageCreateEvent) {
        val message = event.message
        if (message.content.equals(JavaCord.commandPrefix + " online", ignoreCase = true)) {
            event.channel.sendMessage(":tada: " + JavaCord.serverName + " World:" + GameConstants.WORLD + " is Online! :tada:")
        }
    }
}